package.json

```diff
@@ -1,37 +1,87 @@
 {
   "name": "testcafe-reporter-isaac",
-  "version": "0.1.0",
+  "version": "0.0.0-develop",
+  "author": "ISAAC E-commerce Solutions BV",
+  "license": "MIT",
   "description": "An Allure reporter plugin for TestCafé. Used to generate test reports for e2e-tests.",
-  "main": "index.js",
+  "keywords": [
+    "testcafé",
+    "allure",
+    "integration"
+  ],
+  "main": "index.ts",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/isaaceindhoven/testcafe-reporter-isaac.git"
   },
-  "author": "ISAAC E-commerce Solutions BV",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/isaaceindhoven/testcafe-reporter-isaac/issues"
   },
   "homepage": "https://github.com/isaaceindhoven/testcafe-reporter-isaac#readme",
   "scripts": {
+    "build": "tsc",
+    "test": "npm run test:lint && npm run test:unit",
+    "test:lint": "eslint {src,tests}/**/*.{ts,js}",
```

Comment:
The `test:lint`commands fail on my machine with the following error:
```
╰─ npm run test:lint

> testcafe-reporter-isaac@0.1.0 test:lint /Users/maartenvanhoof/Sites/testcafe-reporter-isaac
> eslint {src,tests}/**/*.{ts,js}


Oops! Something went wrong! :(

ESLint: 6.8.0.

No files matching the pattern "src/**/*.js" were found.
Please check for typing mistakes in the pattern.
```

Might I suggest following command: `eslint --ext .ts,.js {src,tests}/`

Run ESLint on all files with a `.ts` or `.js` extension in the `src` or `tests` folders
More info: https://eslint.org/docs/user-guide/command-line-interface#ext

Reply:
`eslint --ext .ts,.js {src,tests}/` does not seem to work on my pc, after the feedback in the retrospective, I went with the `.gitignore` methode:
`eslint --ext .js,.ts --ignore-pattern !.* --ignore-path .gitignore .`

A secondary "problem" occurred when trying to enable linting on the configuration files. On default, EsLint ignores files prepended with a '.', to enable this ignore has to be ignored... by adding `--ignore-pattern !.*` to the command.
Because EsLint looks at the tsconfig too for included files, the following has to be included in the `tsconfig.test.json` file to add the config javascript files to the linting scope.
```
    "include": [
        // Lint all config files in the project. 
        "**/.*"
    ]
```